app/models/null_venue.rb

```diff
@@ -0,0 +1,3 @@
+class NullVenue < Venue
```

Comment:
The null objects I've encountered haven't sublcassed anything. It's not necessarily a bad thing, but it could have surprising consequences. E.g. some functionality is added to `Venue` and not overridden here and that causes some issues. That's hypothetical, so maybe not worth worrying about.

For brevity, the way I've seen them before would be

```
class NullVenue
  def image_url; end
  # whatever other methods need to added
end
```